tgui/packages/tgui/interfaces/CloningConsole.js

```diff
@@ -0,0 +1,398 @@
+    completion,
+    meatLevels,
+    scannerLocked,
+    occupantScanned,
+    scannerOccupied,
+    scannerGone,
+    podGone,
+  } = data;
+
+  return (
+    <Section>
+      <LabeledList>
+        <LabeledList.Item label="Completion">
+          {!podGone && (
+            <ProgressBar
+              value={completion}
+              maxValue={100}
+              minValue={0}
+              ranges={{
+                good: [90, Infinity],
+                average: [25, 90],
+                bad: [-Infinity, 25],
+              }} />
+          )}
+          {!!podGone && (
+            "No Pod Detected"
+          )}
+        </LabeledList.Item>
+        <LabeledList.Item label="Biomatter">
+          {!podGone && (
+            <ProgressBar
+              value={meatLevels}
+              maxValue={100}
+              minValue={0}
+              ranges={{
+                good: [50, 100],
+                average: [25, 50],
+                bad: [0, 25],
+              }} />
+          )}
+          {!!podGone && (
+            "No Pod Detected"
+          )}
+        </LabeledList.Item>
+      </LabeledList>
+      <Box pt={2}>
+        <Button
+          width={7}
+          icon="dna"
+          align={"center"}
+          color={(occupantScanned ? "average" : (scannerGone ? "bad" : "good"))}
+          disabled={occupantScanned | scannerGone}
+          onClick={() => act("scan")}>
+          {(occupantScanned ? "Scanned" : (scannerGone ? "No Scanner" : "Scan"))}
+        </Button>
+        <Button
+          width={7}
+          icon={scannerLocked ? "unlock" : "lock-open"}
+          align={"center"}
+          disabled={!scannerOccupied}
+          color={scannerLocked ? "bad" : "good"}
+          onClick={() => act("toggleLock")}>
+          {scannerLocked ? "Locked" : "Unlocked"}
+        </Button>
+      </Box>
+    </Section>
+  );
+};
+
+const Records = (props, context) => {
+  const { act, data } = useBackend(context);
+  const records = data.cloneRecords || [];
+  const {
+    disk,
+    podGone,
+    diskReadOnly,
+  } = data;
+  const [
+    deletionTarget,
+    setDeletionTarget,
+  ] = useLocalState(context, 'deletionTarget', '');
+
+  return (
+    <Section title="Records">
+      <Table>
+        <Table.Row>
+          <Table.Cell bold collapsing textAlign="center">
+            Name
+          </Table.Cell>
+          <Table.Cell collapsing />
+          <Table.Cell bold textAlign="center">
+            Damage
+          </Table.Cell>
+          <Table.Cell bold textAlign="center">
+            Actions
+          </Table.Cell>
+        </Table.Row>
+        {records.map(record => (
+          <Table.Row key={record.name}>
+            <Table.Cell collapsing textAlign="center">
+              <Box
+                position="relative">
+                {record.id}-{truncate(record.name, 15)}
+                {/* shorten down that name so it doesn"t break the damn gui */}
+                {record.name.length > 20 && (
```

Comment:
why is one 15 and the other 20

Reply:
Forgot to change it 
![image](https://user-images.githubusercontent.com/10260415/93729713-9b3bce00-fbbd-11ea-853f-1dc3d5e33eb9.png)